PowerEditor/installer/APIs/autoit.xml

```diff
@@ -0,0 +1,16099 @@
+<?xml version="1.0" encoding="Windows-1252" ?>
```

Comment:
Is there are reason not to use utf-8 as encoding?

Reply:
IMO, most of API are in English, so ANSI is OK